_data/projects/CodeYourFuture-SDC.yml

```
@@ -0,0 +1,15 @@
+name: Code Your Future SDC
+desc: Code Your Future's main Software Development Course, a PERN web development programme for refugees and other people excluded from education
+site: https://syllabus.codeyourfuture.io/
+tags:
+  - javascript
+  - react
```

```suggestion
  - react.js
```

making changes from bot suggestions